Solutions/Onapsis Integration/Data/Solution_Onapsis.json

```diff
@@ -0,0 +1,29 @@
+{
+    "Name": "Onapsis Integration",
+    "Author": "Onapsis",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/OnapsisBTP.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "Onapsis Integration provides the Onapsis RISE logs to Microsoft Sentinel, allowing SOC teams to ingest, monitor, and hunt across Onapsis data. This integration enhances security by enabling faster detection, investigation, and mitigation of risks within Onapsis RISE environments.",
+    "WorkbookDescription": [],
+    "Workbooks": [],
+    "Analytic Rules": [
```

Comment:
empty the array or migrate and adjust the analytics rules from your legacy onapsis solution folder